styles/style.css

```diff
@@ -0,0 +1,148 @@
+a{
+    color: bisque;
+    font-size: 2rem;
+}
+
+.about{
+    position: absolute;
+    color: bisque;
+    font-size: 1.4rem;
+    background-color:rgb(21, 11, 128);
+    border-radius: 25px;
+    text-align: center;
+    top: 350px;
+    left: 40px; 
+    padding-top: 15px;
+    padding-bottom: 15px;
+    padding-left: 7px;
+    padding-right: 7px;
+}
+
+.about-2{
+    position: absolute;
+    color: bisque;
+    font-size: 1.4rem;
+    background-color:rgb(21, 11, 128);
+    border-radius: 25px;
+    top: 350px;
+    text-align: center;
+    left: 1220px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    padding-left: 7px;
+    padding-right: 7px;
+}
```

Comment:
A lot of these properties are exactly the same as .about-2, which makes me think that it'd be a good idea to have a general `.about` class and then to target the second `.about` paragraph using the pseudo-class `:last-child` or `:nth-child(2)`. So that would be something like this:

```css
.about{
    position: absolute;
    color: bisque;
    font-size: 1.4rem;
    background-color:rgb(21, 11, 128);
    border-radius: 25px;
    top: 350px;
    text-align: center;
    left: 40px;
    padding-top: 15px;
    padding-bottom: 15px;
    padding-left: 7px;
    padding-right: 7px;
}

.about:nth-child(2){
    left: 1220px;
}
```

Reply:
This applies to the `.about:hover` rules too.